sites/all/themes/orwell/sass/component/_author-quote.scss

```diff
@@ -0,0 +1,41 @@
+@import 'bootstrap/bootstrap';
+
+.author-quote a {
+  text-decoration: none;
+  color: black;
+}
+
+.author-quote__wrapper {
+  // To anchor the icons.
+  position: relative;
+  padding: 40px;
```

Comment:
why use px here, if everything else seems to use EM/REM?

Reply:
Also the next PR.